tests/test_report_data/digitrec_report.xml

```diff
@@ -0,0 +1,1420 @@
+<profile>
```

Comment:
I would suggest removing these attached xml and json files, since they can be generated after running HLS. To add test cases, it would be better to give an expected report output and compare your generated report with that one.

Reply:
Currently the report is being displayed as Pandas dataframe object formatted with `tabulate` library. I can convert the table into a list of strings and do a simple string comparison if that is more useful than having XML files as you suggested.

Reply:
Yeah, I think a simple string comparison is okay, or maybe just checking some important metrics (e.g. latency, II) is enough.

Reply:
Actually sorry. The purpose of having all these different XML files (i.e., static test set) instead of just string comparison (i.e., dynamic test set) was because the test suite could not run HLS dynamically with pytest. I unfortunately do not recall the specific reason as to why this was the case. 

@seanlatias Would you be able to comment on this in more detail if you have time?

Reply:
Okay, then just keep those test files.